hax/hax/handler.py

```diff
@@ -158,6 +158,29 @@ def _update_process_status(self, p: WorkPlanner, motr: Motr,
             except Exception as e:
                 LOG.warning("Send event failed due to '%s'", e)
 
+    def cache_process_event(self, state: HAState) -> None:
+
+        def _to_json() -> str:
+            data = {'fid': str(state.fid),
+                    'state': str(state.status.name)}
+            return dump_json(data)
+
+        # cache the process state updates in EQ
+        payload = _to_json()
+        LOG.debug('Process-state-update event JSON: %s', payload)
+        res = self.eq_publisher.publish_no_duplicate(
+            'process-state-update', payload, str(state.fid))
+        if res:
```

Comment:
I think following debug can be removed or replaced with the above debug log, `LOG.debug('Process-state-update event JSON: %s', payload)`

Reply:
combined it to 1 log. Done